### PR TITLE
[match-pattern] JSRS Migration

### PR DESCRIPTION
**Pull request by fdsgusmao**

### CODE REVIEW

**Comment by fdsgusmao** [comment]
### Goals verified:

1. FindBusiness
    - Find a business

2. CallBusiness
    - Call business

3. NavigateThere
    - Navigate there

4. Business#rating
    - What is the rating?

5. Business#categories
    - What are the categories?

**NO ERRORS:** Migration tool migrated the code successfully

- Warnings:0
- Errors:0
- Stage-1 Deprecations:0
- Stage-2 Deprecations:0

image

File: match-pattern/capsule.bxb [hunk 1]
```diff
@@ -5,11 +5,13 @@ capsule {
   targets {
     target (bixby-mobile-en-US)
   }
-  runtime-version (2)
+  runtime-version (2) {
```
Comment (kangawoo44) [inline match-pattern/capsule.bxb:8]: let's make sure we bring up the runtime version to the latest (9) for all sample capsules going forward.

Comment (kangawoo44) [reply match-pattern/capsule.bxb:8]: can you re-run the tests after the update?

**Review by kangawoo44 — APPROVED** [review]
approved once the testing is done on runtimee 9

**Review by abapayeViv — APPROVED** [review]
👍

**Pull request merged**